deployment/hasura/metadata/actions.graphql

```diff
@@ -148,13 +149,18 @@ type SchedulingResponse {
   reason: SchedulingFailureReason
 }
 
-type SchedulingDslTypescriptResponse {
-  status: SchedulingDslTypescriptResponseStatus!
+type DslTypescriptResponse {
+  status: DslTypescriptResponseStatus!
   typescript: String
   typescriptFiles: [TypescriptFile]
   reason: String
 }
 
+enum DslTypescriptResponseStatus {
+  success
+  failure
+}
+
 type TypescriptFile {
   filename: String,
```

Comment:
What would it take to change `contents` to `content`, and `filename` to `filePath`? This would allows us to pass a `TypescriptFile[]` directly to Monaco :)

Reply:
As a curious observer -- is there a way (maybe interface subtyping?) to formalize the alignment you want with Monaco, so that the intent to use this data with Monaco is clear to both us humans and the compiler? (Or, once we're passing things to Monaco, will the structural type system reliably give us an error regardless?)

Reply:
Yeah another option I'm thinking is to just alias the query on the client. But I'm all for making the type alignment more formal.

Reply:
Yeah I'll make that change, but you'll have to update the scheduling Dsl request too. They return the same struct.

Reply:
There is a great package @graphql-codegen to manage the typescript graphql typing interface. @camargo likely something you might want to use long term to avoid issues

Reply:
Yup good recommendation. Have used it on projects before and it's nice.

Reply:
Oh, right the UI doesn't use this field yet. @camargo Since you're making that change, can I go ahead and remove the `DslTypescriptResponse.typescript` field while we're at it?

Reply:
Yes no problem on my end

Reply:
done